### PR TITLE
fix: footer styles, adjust error message on the not found page

### PR DESCRIPTION
**Pull request by mrkldshv**
* Fixes footer position making it stick to bottom
* Adjust error message on the not found page

Before: 
Screen Shot 2022-05-06 at 22 38 40
After:
Screen Shot 2022-05-07 at 23 33 48

### CODE REVIEW

File: middleware/notFound.tsx [hunk 1]
```diff
@@ -15,15 +15,11 @@ export const handleNotFound: Middleware = async (ctx, next) => {
       const page = renderSSR(
         <App>
           <ErrorBody title="Not Found">
-            The requested URL ({`<code>"${ctx.request.url}"</code>`}) was not
-            found.
+            The requested URL <code>{ctx.request.url.href}</code> was not found.
```
Comment (mrkldshv) [inline middleware/notFound.tsx:18]: Not sure about this change, but I guess we shouldn't display `code` tag in the error message.

File: components/app.tsx [hunk 2]
```diff
@@ -32,7 +32,7 @@ function Footer() {
   return (
     <footer
       class={tw
-        `mt-20 max-w-screen-xl mx-auto py-12 px-4 overflow-hidden sm:px-6 lg:px-8`}
+        `sticky top-full mt-20 max-w-screen-xl mx-auto py-12 px-4 overflow-hidden sm:px-6 lg:px-8`}
```
Comment (kt3k) [inline components/app.tsx:35]: 👍

**Review by kt3k — APPROVED** [review]
LGTM, thanks!

**Comment by mrkldshv** [comment]
@kt3k thank you for review!

**Pull request merged**